scripts/check_parse_genotype_files.py

```diff
@@ -0,0 +1,92 @@
+#!/bin/env python
+
+"""Check for the presence of genotype files and update Charon accordingly."""
+import argparse
+import glob
+import os
+import re
+import time
+
+from ngi_pipeline.database.classes import CharonSession, CharonError
+from ngi_pipeline.log.loggers import minimal_logger
+from ngi_pipeline.utils.charon import find_projects_from_samples
+from ngi_pipeline.utils.classes import with_ngi_config
+from ngi_pipeline.utils.parsers import parse_samples_from_vcf
+
+LOG = minimal_logger(os.path.basename(__file__))
+
+
+# This may need to be changed
+GENOTYPE_FILE_RE = re.compile(r'\d{4}\S*.vcf')
```

Comment:
I don't know exactly what the format will be, but we can update this when we determine that. Right now it picks up any file that starts with four numbers and ends with ".vcf"